app/javascript/src/components/Dashboard/Notes/DropMenu.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const DropMenu = ({ onClick }) => {
+  const { Menu, MenuItem } = Dropdown;
```

Comment:
Keep this outside the component so that we don't run this statement during re-rendering of this component.

Reply:
Thanks for pointing this out😀! I never thought of the effect of re-render on variable declaration inside a component. I will move it outside👍.